modules/core/src/views/view-manager.js

```diff
@@ -244,7 +240,7 @@ export default class ViewManager {
         // TODO - check for removed view ids?
         if (controller) {
           controller.setProps(
-            Object.assign({}, viewState, {
+            Object.assign({}, view.controller, viewState, {
```

Comment:
Ditto

Reply:
`view.controller` is a normalized object (or null).